src/backend/main_window.ts

```diff
@@ -66,12 +70,16 @@ export const createMainWindow = () => {
     }
   }
 
+  // setup the titlebar main process
+  setupTitlebar()
```

Comment:
I think it's important to make this optional

- with this custom titlebar we lose the features the window manager shows when you right click the title bar for example
- users may have their own window decoration style/personalizations they want to use instead (like text alignment, font, icons, etc)